benchmarks/shallowEqual.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+const Benchmark = require('benchmark');
+
+const suite = new Benchmark.Suite();
+
+function shallowEvery(a, b) {
+  if (a.length !== b.length) {
+    return false;
+  }
+
+  return a.every((e, i) => b[i] === e);
+}
+
+function shallowFor(a, b) {
+  if (a.length !== b.length) {
+    return false;
+  }
+
+  for (let i = 0; i < a.length; i++) {
+    if (a[i] !== b[i]) {
+      return false;
+    }
+  }
+  return true;
+}
+
+let a = {};
+let b = {};
+
+let listA = [a, b, a, b, a, b, a, b, a, b, a, b, {}, {}, {}];
```

Comment:
A more real-world benchmark would have fewer arguments to reflect what you might pass a memoized function (maybe 1-4?)